src/python/pants/python/python_setup.py

```diff
@@ -100,13 +100,14 @@ def register_options(cls, register):
         register(
             "--resolver-jobs",
             type=int,
-            default=multiprocessing.cpu_count() // 2,
+            default=CPU_COUNT // 2,
```

Comment:
Not for this PR, but it seems to me the verbage is now false when a repo is setup with a "lockfile". In that case we should be resolving once and we should be bottlenecking on that resolve and so we could be using full Pex default parallelism (we'd still do tool resolves separately, but ... maybe uninmportant?).

Reply:
Agreed, unless you're doing multiple tool resolves at the same time 🤔